File: Constructorio_Net/src/models/ResultGroup.java
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

/**
 * Constructor.io Item Group
 */
*/
namespace Constructorio_NET
{
    public class ResultGroup {

        [JsonPropertyName("display_name")]
        public String displayName;

        [JsonPropertyName("group_id")]
        public String groupId;

        [JsonPropertyName("path")]
        public String path;

    }
}

File: Constructorio_NET/src/models/ResultGroup.cs
using System;
using Newtonsoft.Json;

/**
 * Constructor.io Item Group
 */
namespace Constructorio_NET
{
    public class ResultGroup {

        [JsonProperty("display_name")]
        public String DisplayName { get; set; }

        [JsonProperty("group_id")]
        public String GroupId { get; set; }

        [JsonProperty("path")]
        public String Path { get; set; }

    }
}